src/concourse/pipelines/infrastructure/concourse/pipeline.py

```diff
@@ -0,0 +1,216 @@
+from concourse.lib.constants import REGISTRY_IMAGE
```

Comment:
Don't think I'm very clear on what's going on in this file

Reply:
The idea of this module is to store values that are used frequently and are immutable. For instance `"registry-image" is a string we will be using often and this way we can avoid accidental typos that lead to enigmatic bugs.